src/components/tile-select/tile-select.pw.tsx

```diff
@@ -0,0 +1,610 @@
+    });
+
+    test(`should check actionButtonAdornment iconType for TileSelect component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<WithActionButtonAdornment />);
+
+      const iconElement = getComponent(page, "icon").nth(0);
+      await iconElement.isVisible();
+      await expect(iconElement).toHaveAttribute("type", "info");
+    });
+  });
+
+  test.describe(`should render TileSelect component and check events`, () => {
+    test(`should call onChange callback when a click event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <TileSelectGroupComponent
+          onChange={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      const onChangeElement = inputElement(page).first();
+      await onChangeElement.click();
+      expect(callbackCount).toBe(1);
+    });
+
+    test(`should call onBlur callback when a blur event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <TileSelectGroupComponent
+          onBlur={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      const blurElement = inputElement(page).first();
+      await blurElement.click();
+      await blurElement.blur();
+      expect(callbackCount).toEqual(1);
+    });
+
+    test(`should call onFocus callback when a focus event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <TileSelect
+          onFocus={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      const onFocusElement = inputElement(page);
+      await onFocusElement.focus();
+      await expect(callbackCount).toEqual(1);
+    });
+
+    test(`should call onChange callback when a click event is triggered for TileSelect component`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <TileSelect
+          onChange={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      const onChangeElement = inputElement(page).first();
+      await onChangeElement.click();
+      expect(callbackCount).toBe(1);
+    });
+
+    test(`should call onBlur callback when a blur event is triggered for TileSelect component`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <TileSelect
+          onBlur={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      const blurElement = inputElement(page).first();
+      await blurElement.click();
+      await blurElement.blur();
+      expect(callbackCount).toEqual(1);
+    });
+  });
+
+  test.describe(`Accessibility tests for Tile Select`, () => {
+    // FE-4683
+    test.skip(`should pass accessibility tests for Default story`, async ({
```

Comment:
**suggestion(non-blocking):** These will need changing because they aren't technically stories anymore. 

```suggestion
    test.skip(`should pass accessibility tests for Default example`, async ({
```